### PR TITLE
feat(trace-explorer): Use new SearchQueryBuilder

### PR DESCRIPTION
**Pull request by Zylphrex**
This switches the trace explorer to use the new SearchQueryBuilder.

### CODE REVIEW

File: static/app/views/traces/content.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ const DEFAULT_PER_PAGE = 50;
 export default function Wrapper(props) {
   const organization = useOrganization();
 
-  if (organization.features.includes('xvisibility-explore-view')) {
+  if (organization.features.includes('visibility-explore-view')) {
```
Comment (malwilley) [inline static/app/views/traces/content.tsx:42]: Was this meant to be part of these changes?

File: static/app/components/performance/spanSearchQueryBuilder.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  const supportedTags = useSpanFieldSupportedTags({
+    projects: projects ?? selection.projects,
+  });
+
+  const filterTags: TagCollection = useMemo(() => {
+    return {...supportedTags};
+  }, [supportedTags]);
+
+  const filterKeySections = useMemo(() => {
+    return [];
```
Comment (malwilley) [inline static/app/components/performance/spanSearchQueryBuilder.tsx:53]: Should this be filled out? Categories would help the user find the right keys

File: static/app/components/performance/spanSearchQueryBuilder.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  datetime,
+  onSearch,
+  placeholder,
+  projects,
+}: SpanSearchQueryBuilderProps) {
+  const api = useApi();
+  const organization = useOrganization();
+  const {selection} = usePageFilters();
+
+  const placeholderText = useMemo(() => {
```
Comment (malwilley) [inline static/app/components/performance/spanSearchQueryBuilder.tsx:40]: nit: memo not necessary for string props

File: static/app/views/traces/content.tsx (hunk 1 above)
Comment (Zylphrex) [reply static/app/views/traces/content.tsx:42]: Yes, just a typo that I had and doesn't affect any customers

File: static/app/components/performance/spanSearchQueryBuilder.tsx (hunk 2 above)
Comment (Zylphrex) [reply static/app/components/performance/spanSearchQueryBuilder.tsx:53]: Yes, just starting without it and will fill it in in a followup

**Review by malwilley — APPROVED** [review]

File: static/app/components/performance/spanSearchQueryBuilder.tsx (hunk 3 above)
Comment (Zylphrex) [reply static/app/components/performance/spanSearchQueryBuilder.tsx:40]: will address in the follow up

**Pull request merged**